cmd/utils.go

```diff
@@ -314,6 +314,9 @@ func getVolumeArgs(config *RootCommandConfig) ([]string, error) {
 			mappedMount = strings.Replace(mount, "~", homeDir, 1)
 			overridden = homeDirOverridden
 		} else {
+			if strings.HasPrefix(mount, ".:") {
```

Comment:
Maybe another options would be to use `$(pwd)`?

Reply:
@EnriqueL8 not following since that's a shell expansion and I'm not too familiar with Go.   Do you mean something like: `https://golang.org/pkg/os/#Getwd` ?